src/addon.cc

```diff
@@ -7,26 +7,54 @@
 using namespace v8;
 
 void init(Local<Object> exports) {
+  v8::Isolate* isolate = v8::Isolate::GetCurrent();
+  v8::Local<v8::Context> context = isolate->GetCurrentContext();
+  v8::HandleScope handle_scope(isolate);
   Nan::Set(exports,
     Nan::New<String>("version").ToLocalChecked(),
     Nan::New<String>(VERSION).ToLocalChecked());
 
-  Nan::Set(exports,
-    Nan::New<String>("verify").ToLocalChecked(),
-    Nan::New<FunctionTemplate>(verify)->GetFunction());
+  {
+    v8::Local<v8::Function> method;
+    Nan::New<FunctionTemplate>(verify)->GetFunction(context).ToLocal<v8::Function>(&method);
```

Comment:
1) `ToLocal` does not check if monad is empty or not. Better to use `ToLocalChecked()`
2) It is a little bit stinky, to use lexical scopes for each set.